pkg/remediator/reconcile/worker_test.go

```diff
@@ -85,7 +79,7 @@ func TestWorker_Run_Remediates(t *testing.T) {
 		}
 	}
 
-	d := makeDeclared(t, declaredObjs...)
+	d := makeDeclared(t, "commit-unused", declaredObjs...)
```

Comment:
You may want to define a constant for "commit-unused", considering it is being used multiple times here.

Reply:
Surprisingly, the linter didn't complain. Must not have hit the threshold for extracting constants.

Reply:
Used a UUID generator instead of a constant. Now at least we can see if it ends up being accidentally tested.